code/game/objects/items/weapons/weaponry.dm

```diff
@@ -286,3 +286,17 @@
 	icon_state = "tailwhip"
 	origin_tech = "engineering=3;combat=3;biotech=3"
 	needs_permit = 0
+
+/obj/item/weapon/melee/skateboard
+	name = "skateboard"
+	desc = "A skateboard. It can be placed on its wheels and ridden, or used as a strong weapon."
+	icon_state = "skateboard"
+	item_state = "skateboard"
+	force = 12
+	throwforce = 4
+	w_class = 5.0
+	attack_verb = list("smacked", "whacked", "slammed", "smashed")
+
+/obj/item/weapon/melee/skateboard/attack_self(mob/user)
+	new /obj/vehicle/scooter/skateboard(get_turf(user))
+	qdel(src)

```

Comment:
Why not have the object stored inside the vehicle instead of constantly deleting both?